app/components/ui/atoms/ManuscriptStatus.js

```diff
@@ -2,12 +2,16 @@ import React from 'react'
 import styled from 'styled-components'
 import { Box } from 'grid-styled'
 import PropTypes from 'prop-types'
+import { th } from '@pubsweet/ui-toolkit'
+import media from '../../global/layout/media'
 
 const StatusBox = styled(Box)`
```

Comment:
Given that this styling is very specific to the use of `ManuscriptStatus` within `DashboardListItem` (as opposed to it being re-usable in multiple contexts), I'm not sure that having `ManuscriptStatus` as its own separate atom is particularly necessary? If anything, it just means that a developer now needs to look in 2 files to know how `DashboardListItem` is built.

What do you think of putting this inside `DashboardListItem.js`?

Reply:
I've got no reason other than personal choice to argue against this. I find working on smaller files easier as it means theres less helper functions to scroll through to find what you're after and with cmd - click it makes it simple enough to navigate to imported files. Thats my opinion though, so I wouldn't disagree with making this change if the consensus was there.

Reply:
My vote is the path of least resistance -- so it looks like I'm siding with "if it aint broke don't fix it"

Reply:
Yeah I'm happy to chalk this up to personal preference. If anyone actually finds it an unnecessary extra piece in future, they're welcome to amalgamate the two. There's little point in changing it now, if it's not a widely held opinion :+1: